datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/InboxController.java

```diff
@@ -60,6 +75,12 @@ public void writeToInbox(@RequestHeader Set<String> users,
      */
     @SneakyThrows
     @GetMapping(value = "/{path:.*}", produces = APPLICATION_OCTET_STREAM_VALUE)
+    @ApiOperation("Read document from inbox")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Document was successfully read"),
+            @ApiResponse(code = 403, message = "Access denied"),
```

Comment:
403 has default message, not necessary to add it

Reply:
removed